.github/workflows/pull_request.yml

```diff
@@ -23,17 +23,14 @@ jobs:
       linux_nightly_6_0_arguments_override: "--explicit-target-dependency-import-check error"
       linux_nightly_main_arguments_override: "--explicit-target-dependency-import-check error"
 
+  benchmarks:
+    name: Benchmarks
+    # Workaround https://github.com/nektos/act/issues/1875
+    uses: ./.github/workflows/benchmarks.yml
+    with:
+      benchmark_package_path: "Benchmarks"
+
   cxx-interop:
     name: Cxx interop
-    uses: apple/swift-nio/.github/workflows/swift_matrix.yml@main
-    with:
-      name: "Cxx interop"
-      matrix_linux_command: "apt-get update -y -q && apt-get install -y -q jq && apt-get -y install libsasl2-dev && curl -s https://raw.githubusercontent.com/apple/swift-nio/main/scripts/check-cxx-interop-compatibility.sh | bash"
-      matrix_linux_5_9_enabled: true
-      matrix_linux_5_10_enabled: true
-      matrix_linux_6_0_enabled: true
-      matrix_linux_nightly_6_0_enabled: true
-      matrix_linux_nightly_main_enabled: true
-      matrix_windows_6_0_enabled: false
-      matrix_windows_nightly_6_0_enabled: false
-      matrix_windows_nightly_main_enabled: false
+    # Workaround https://github.com/nektos/act/issues/1875
```

Comment:
```suggestion
```

This comment is not relevant here, it's to do with chained workflow imports, it's copy-pasta.